scorecard/scorecard/scorecard.py

```diff
@@ -65,34 +59,20 @@
 def index():
     user_data, unassigned, urgent_issues, updated = get_users()
 
-    random_user = random.choice(users)
+    def functional_shuffle(x):
+        x = list(x)
+        random.shuffle(x)
+        return x
 
+    component_users_random = {c: functional_shuffle(us) for c, us in component_users.items()}
```

Comment:
Rather than returning the full list, why not shuffle and then return the first element?

Reply:
No reason.  I can change it if you prefer.

Reply:
Changed it to include one user.  Also obviates @danking's comment.